pkg/backuphelpers/backupvars.go

```diff
@@ -55,6 +56,34 @@ func (b *BackupConfig) SetBackupSpec(spec *backupv1alpha1.EtcdBackupSpec) {
 	}
 }
 
+func (b *BackupConfig) ArgList() []string {
```

Comment:
The templating changes for etcd_assets and the targetconfigcontroller look good but is this related to that?
I can see the corresponding test for this but not sure where we're invoking `ArgList()`?
Wasn't sure if this is meant for the template FuncMap?

(Or my brain is mush right now and I can't grep it :P)

Reply:
right here: https://github.com/openshift/cluster-etcd-operator/pull/1346/files#diff-0af15f182aea9898e784fc589175ef4c2411c9391bb5a1f41327ba7c3e2172f5R226